packages/rum/test/specs/apm-base.spec.js

```diff
@@ -149,9 +151,31 @@ describe('ApmBase', function () {
     })
   })
 
+  it('should observe click event if eventtarget instrumentation is not disabled', () => {
+    apmBase.init({ serviceName, serverUrl })
+
+    document.body.click()
+
+    const tr = apmBase.getCurrentTransaction()
+    expect(tr.name).toBe('Click - body')
+  })
+
+  it('should not observe click event if eventtarget instrumentation is disabled', () => {
+    apmBase.init({
+      serviceName,
+      serverUrl,
+      disableInstrumentations: [EVENT_TARGET]
```

Comment:
we might have to change the naming to click. Feels weird to have it `eventtarget`. 

The disableInstrumentations can be monkeypatched to support both names and we can depreacte `eventtarget` later when doing `6.x`.

Reply:
Good point!. I was thinking that maybe what we can do is:

- Create the new name.
- Keep supporting the old one (like you are saying), but in the release notes and in the [documentation](https://www.elastic.co/guide/en/apm/agent/rum-js/current/configuration.html#disable-instrumentations) that it's considered **deprecated** and that in a future version will be removed

Will update the [6.x issue](https://github.com/elastic/apm-agent-rum-js/issues/772) too